src/main/java/org/broadinstitute/hellbender/utils/fasta/CachingIndexedFastaSequenceFile.java

```diff
@@ -159,7 +160,7 @@ public static CachingIndexedFastaSequenceFile checkAndCreate(final File fastaFil
 
         // determine the name for the dict file
         final String fastaExt = fastaFile.getAbsolutePath().endsWith("fa") ? "\\.fa$" : "\\.fasta$";
-        final File dictFile = new File(fastaFile.getAbsolutePath().replaceAll(fastaExt, ".dict"));
+        final File dictFile = new File(fastaFile.getAbsolutePath().replaceAll(fastaExt, IOUtil.DICT_FILE_EXTENSION));
```

Comment:
I just kept the previous behaviour, but using the `IOUtil.DICT_FILE_EXTENSION` for keeping in sync with htsjdk.

Nevertheless, I added a TODO to use the htsjdk method to find the default dictionary name for the reference. Than may be simpler, to keep also in sync with `CreateSequenceDictionary`.